### DISCUSSION TITLE
Collect technical questions and solutions via Github.

### DISCUSSION THREAD

**Discussion by alejandroRbit** [Magickbase/shaping#51] [Ideas]
I have noticed that various members of the community are collecting Questions and Answers/Solutions that often occur on Discord and other platforms. I think it is a very good idea and has a number of important advantages:

- As Phroi rightly comments #https://github.com/ckb-js/lumos/issues/513#issue-1709571146, GitHub issues are SEO friendly.
- Attract the community to collaborate on Github.
- Facilitate developer support.
- Facilitates learning for new community developers.
- Improves accessibility to information.
- It's something the community can help with if they have a guideline.

Here some **examples**:

**Using Q/A from the discussion section:**
How to translate godwoken_short_address to eth_address? https://github.com/godwokenrises/godwoken/discussions/1069

**Using an issue:** 
How to import the next Lumos version? https://github.com/ckb-js/lumos/issues/513

**Using Nervos Talk**: 
What’s the max size and max cycles of a CKB block? https://talk.nervos.org/t/whats-the-max-size-and-max-cycles-of-a-ckb-block/7166

This comes up from a series of questions from a developer on the Magickbase discord:

https://discord.com/channels/956765352514183188/959384192859389983/1108895398803800134

**Comment by Keith-CY** [comment]
Sure, we encourage members of the community to propose their ideas/questions on GitHub, which is actively maintained by our team. Actually, GitHub is **the primary channel** we're in, while other channels are for non-technical members.

By doing so, these discussions can be archived and accessible to the community.

These discussions/solutions will also be concluded and subscribed to by our troubleshooting page(https://github.com/Magickbase/neuron-public-issues/issues/99) to utilize the power of algolia(https://www.algolia.com/), which is more powerful than the search of GitHub/Google, and easy to visit for non-technical members.

Besides, Q&A on other channels will be recorded on GitHub too, if they are frequently asked, to make the library(GitHub issues) robust. And all these issues will be referenced if similar questions appear on other channels.
